nupic/algorithms/KNNClassifier.capnp

```diff
@@ -0,0 +1,41 @@
+@0x99e493bfb8fbad94;
+
+# Next ID: ???
+struct KNNClassifierProto {
+  version @0 :UInt16;
+  verbosity @1 :UInt8;
```

Comment:
I don't think we should force everything to be 32 bits. The verbosity makes sense as UInt8.

The version can be dropped for now since it isn't needed with capnp